core/arch/arm/kernel/unwind_arm64.c

```diff
@@ -34,13 +34,21 @@
 #include <string.h>
 #include <trace.h>
 
-bool unwind_stack(struct unwind_state *frame)
+bool unwind_stack_arm64(struct unwind_state_arm64 *frame, uaddr_t stack,
+		      size_t stack_size)
 {
 	uint64_t fp;
 
 	fp = frame->fp;
-	if (!thread_addr_is_in_stack(fp))
-		return false;
+	if (stack && stack_size) {
```

Comment:
I'd prefer if `stack` and `stack_size` always where correct, that way we could avoid this special case.